### PR TITLE
Updating utils links

### PR DESCRIPTION
**Pull request by benstokoe**
Utils were put into src folder

### CODE REVIEW

**Comment by cpsubrian** [comment]
Haha noticed this and low-and-behold there is already a PR :+1:

File: README.md [hunk 1]
```diff
@@ -105,12 +105,12 @@ This has many different use cases like:
 * Isomorphism. You save a snapshot that you send from the server to the client and then bootstrap back on the client.
 * Rolling back to previous stable states.
 
-There are also many [utils](/utils) available which interface well with alt:
+There are also many [utils](/src//utils) available which interface well with alt:
```
Comment (goatslacker) [inline README.md]: can you clean up the double slashes?

**Comment by goatslacker** [comment]
Good catch :eyes:

**Comment by benstokoe** [comment]
Thanks for that catch! That's what a quick in work edit gets you!

**Pull request merged**